src/plugins/flow.js

```diff
@@ -1052,9 +1055,10 @@ export default function (instance) {
   instance.extend("parseObjPropValue", function (inner) {
     return function (prop) {
       if (prop.variance) {
-        this.unexpected(prop.variance.start);
+        this.unexpected(prop.start);
```

Comment:
👍 Yup. In this case they are one-in-the-same, but this should definitely be `variancePos` for clarity. We wouldn't need `variancePos` except for static properties on classes, which share the same AST building code.